executor/analyze.go

```diff
@@ -435,3 +442,52 @@ func (e *AnalyzeColumnsExec) buildStats() (hists []*statistics.Histogram, cms []
 	}
 	return hists, cms, nil
 }
+
+func analyzeFastExec(exec *AnalyzeFastExec) statistics.AnalyzeResult {
+	hists, cms, err := exec.buildStats()
+	if err != nil {
+		return statistics.AnalyzeResult{Err: err}
+	}
+	result := statistics.AnalyzeResult{
+		PhysicalTableID: exec.PhysicalTableID,
+		Hist:            hists,
+		Cms:             cms,
+	}
+	hist := hists[0]
+	result.Count = hist.NullCount
```

Comment:
What about exporting `Histogram.totalRowCount`

Reply:
Yeah. It's not correct here. This function needs to distinguish idx result and col result. I'll fix it.

Reply:
Already fixed.